app/views/placements/providers/users/index.html.erb

```diff
@@ -5,6 +5,7 @@
   <h1 class="govuk-heading-l"><%= t(".users") %></h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      <p><%= t(".intro_text") %></p>
```

Comment:
Generally speaking, paragraphs should have the class `govuk-body`.

```suggestion
      <p class="govuk-body"><%= t(".intro_text") %></p>
```

It looks like our CSS is [configured in such a way](https://github.com/DFE-Digital/itt-mentor-services/blob/598cef6bec190a6021994a18f7d1ad2606187327/app/assets/stylesheets/application.scss#L1) that naked `<p>` tags still get the correct styling, but this is not the default behaviour of the GOV.UK Design System.

Best practice is to make sure we apply this class to all page content (as we [already do](https://github.com/search?q=repo:DFE-Digital/itt-mentor-services%20govuk-body&type=code) elsewhere in this app).

Reply:
(this comment applies for all `<p>` tags added in this PR)